core/utils.py

```diff
@@ -290,18 +292,36 @@ async def send_dm(
 
 
 class ScrollableEmbed:
-    def __init__(self):
-        self.pages = []
+    """Class for making scrollable embeds easy.
+
+    Args:
+        ctx (:class:`discord.ext.commands.Context`): The context for translational purposes.
+        iterable (:class:`Iterable[discord.Embed]`): Iterable which to build the ScrollableEmbed from.
+    """
+
+    def __init__(
+        self, ctx: commands.Context, iterable: Iterable[discord.Embed]
+    ) -> ScrollableEmbed:
+        """[summary]
+
+
+
+        Returns:
+            ScrollableEmbed: [description]
```

Comment:
Hmm, can you clean this docstring, please?

Reply:
Oops I left the autogenerated text there. I deleted the docstring completely as there is no use to it in an `__init__` method